src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.MoveFile.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+public partial class FileSystemWatcher_4000_Tests
+{
+    #region WindowsTests
+
+    [Theory]
+    [InlineData(WatcherChangeTypes.Changed, "changed", false)]
```

Comment:
I could not find a way in the xunit docs to make the tests within a class run in parallel: they have to be in different classes. I tested them in different classes and it does run considerably faster, but it's quite a bit less readable as a result. I'm not convinced it's worth the tradeoff; I'm going to revert to the theories and ```[outerloop]``` the different directory tests.